### PR TITLE
BlenderFoundation.Blender version 3.6.2

### PR DESCRIPTION
**Pull request by unwillinglyawesome**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/117092)

### CODE REVIEW

File: manifests/b/BlenderFoundation/Blender/3.6.2/BlenderFoundation.Blender.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- 3ds
+- fbx
+- glb
+- gltf
+- obj
+- ply
+- stl
+Installers:
+- Architecture: x64
+  InstallerUrl: https://mirror.freedif.org/blender/release/Blender3.6/blender-3.6.2-windows-x64.msi
```
Comment (stephengillie) [inline manifests/b/BlenderFoundation/Blender/3.6.2/BlenderFoundation.Blender.installer.yaml:23]:
```suggestion
  InstallerUrl: https://mirrors.ocf.berkeley.edu/blender/release/Blender3.6/blender-3.6.2-windows-x64.msi
```
Hi @unwillinglyawesome,

This Berkeley.edu link is the InstallerUrl that I get when starting at the PackageUrl (https://www.blender.org) and navigating through the Download links. Where did you find the freedif.org link?

Comment (unwillinglyawesome) [reply manifests/b/BlenderFoundation/Blender/3.6.2/BlenderFoundation.Blender.installer.yaml]: On blender.org, it chooses a random mirror is my guess.

Comment (unwillinglyawesome) [reply manifests/b/BlenderFoundation/Blender/3.6.2/BlenderFoundation.Blender.installer.yaml]: image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**